public/js/main.js

```diff
@@ -243,10 +243,10 @@ function setFileURLs(new_provider) {
         var user = getSafeHighlightedValue(hit._highlightResult.github.user);
         var repo = getSafeHighlightedValue(hit._highlightResult.github.repo);
         githubDetails = '<ul class="list-inline">' +
-          '<li><i class="fa fa-github"></i> <a href="https://github.com/' + hit.github.user + '/' + hit.github.repo + '" target="_blank">' + user + '/' + repo + '</a></li>' +
-          '<li><i class="fa fa-eye"></i> ' + hit.github.subscribers_count + '</li>' +
-          '<li><i class="fa fa-star"></i> ' + hit.github.stargazers_count + '</li>' +
-          '<li><i class="fa fa-code-fork"></i> ' + hit.github.forks + '</li>' +
+          '<li><i class="fab fa-github"></i> <a href="https://github.com/' + hit.github.user + '/' + hit.github.repo + '" target="_blank">' + user + '/' + repo + '</a></li>' +
+          '<li><i class="far fa-eye"></i> ' + hit.github.subscribers_count + '</li>' +
```

Comment:
I wonder if the "Regular" style is okay for us? Looks like it's not part of free plan, I'm not sure, ref: https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use

Reply:
https://fontawesome.com/icons/eye?style=regular the regular eye is free to use :) It's the shim for the fa 4.x eye.

Reply:
hmmm, thanks. Do you have any idea about "Pro Required" on the link I post?

Reply:
Most regular and light icons do fall under pro only, however, as the regular eye is the shim for FA 4.x, it is available for free use.

Reply:
Cool :+1: